integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
Maybe related to https://github.com/dnephin/cobra/blob/fork/command.go#L358-L396 ?

I think the error is coming from https://golang.org/pkg/encoding/csv/, which I think is called from `opts.ListOpts`. I think pflags must have been doing something funky to make this work? Maybe the "add single quotes" patch that we didn't add.

What if you use a `StringSliceVar()` here instead of `ListOps` ?